README.md

```diff
@@ -4,9 +4,6 @@
 **This version of the Conviva Analytics Integration works only with Player Version 7.x.
 The recommended version of the Conviva SDK is 2.146.0.36444.** See [CHANGELOG](CHANGELOG.md) for details.
 
-## Limitations
-This Conviva integration currently does not track pre-roll ads. The analytics session is created after pre-roll ads, before the main content is started. This is implemented according to Conviva's guidelines but is still considered a limitation as the `Exit Before Video Start` metrics cannot be measured. All other metrics are working as expected.
```

Comment:
How did you implement this?

Reply:
Some time ago we changed the way how we initialize the session. The session is initialized `onPlay` and this is triggered also in case of pre-roll ads. So this was no longer a limitation.

Reply:
Nice!